persistence/src/main/resources/AQLFunctions/README.txt

```diff
@@ -0,0 +1,15 @@
+Spline ArangoDB AQL functions are defined in JS files in the following pattern:
```

Comment:
Maybe we should mention the AQL limitations here

Reply:
Let's assume we're the last ones who don't read manuals :)